src/tests/ftest/datamover/posix_subsets.py

```diff
@@ -113,11 +127,15 @@ def run_dm_posix_subsets(self, tool):
         # Each src or dst is a list of params:
         #   [param_type, path, pool, cont]
         for (test_desc, src, dst) in copy_list:
-            self.run_datamover(
+            result = self.run_datamover(
                 test_desc,
                 src[0], src[1], src[2], src[3],
                 dst[0], dst[1], dst[2], dst[3])
             self.read_verify_location(*dst)
+            if self.tool == "FS_COPY":
+                fs_copy_search = re.search(r"Successfully copied to DAOS", result.stdout_text)
+                if not fs_copy_search.group(0) == "Successfully copied to DAOS":
+                    self.fail("Failed to copy to DAOS")
```

Comment:
Careful with this. `re.search` returns `None` if the string isn't found. E.g.
```
>>> import re
>>> my_text = "apples oranges"
>>> search = re.search(r"peaches", my_text)
>>> search.group(0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'group'
```
And if it IS found, then it's not necessary to compare `group(0)`, since it will be what you searched for. To handle the error condition, and simplify the check:
```suggestion
                fs_copy_search = re.search(r"Successfully copied to DAOS", result.stdout_text)
                if not fs_copy_search:
                    self.fail("Failed to copy to DAOS")
```
Or even:
```
                if not re.search(r"Successfully copied to DAOS", result.stdout_text):
                    self.fail("Failed to copy to DAOS")
```
Or...
```
                if not "Successfully copied to DAOS" in result.stdout_text:
                    self.fail("Failed to copy to DAOS")
```